ci/scenarios/vPool/add_remove_vpool_concurrently/main.py

```diff
@@ -32,39 +34,35 @@ class VPoolTester(CIConstants):
     NUMBER_OF_VPOOLS = 3
     LOGGER = Logger('scenario-{0}'.format(TEST_NAME))
 
-    def _initialize_environment(self):
-        """
-        this method sets up or checks environmental prerequisits for testing: storage routers and backends
-        """
-        self.LOGGER.info("Initializing environment for concurrent vpool testing")
-        valid_storagerouters = []
+    def __init__(self):
+        self.LOGGER.info("Initializing concurrent vpool testing.")
+        self.valid_storagerouters = []
         for storagerouter in StoragerouterHelper.get_storagerouters():
             try:
                 RoleValidation.check_required_roles(VPoolSetup.REQUIRED_VPOOL_ROLES, storagerouter.ip, "LOCAL")
-                valid_storagerouters.append(storagerouter)
-                self.LOGGER.info("Added {0} to list of eligible storagerouters".format(storagerouter.ip))
+                self.valid_storagerouters.append(storagerouter)
+                self.LOGGER.info("Added {0} to list of eligible storagerouters.".format(storagerouter.ip))
             except RuntimeError as ex:
-                self.LOGGER.warning("Did not add {0} to list of eligible storagerouters: {1}".format(storagerouter.ip, ex))
+                self.LOGGER.warning("Did not add {0} to list of eligible storagerouters: {1}.".format(storagerouter.ip, ex))
```

Comment:
No need to log this in my opinion
For this test you just want 1 eligible StorageRouter, if you manage to find 1, but any others don't have the required roles, there is no need to log this. Only causes confusion